styles/styles.css

```diff
@@ -909,3 +905,92 @@ main.blue-contract .section.section-with-title p {
     margin: 0 0 80px;
   }
 }
+
+
+/* REDESIGN GLOBAL STYLES */
+:root.redesign-v2 {
+  font-size: 16px;
+}
+
+:root.redesign-v2 strong {
+  font-family: var(--ff-volvo-novum-medium);
+  font-weight: 100;
+}
+
+:root.redesign-v2 .section {
+  padding: 0;
+}
+
+:root.redesign-v2 .section > div {
+  padding: 0 16px;
+  margin: auto;
+}
+
+:root.redesign-v2 .section div.full-width {
+  padding: 0;
+  width: 100%;
+}
+
+.redesign-v2 a.button.primary,
+.redesign-v2 button.primary {
+  display: inline-flex;
+  justify-content: center;
+  gap: 8px;
+  color: var(--c-cta-blue-default);
+  font-feature-settings: 'clig' off, 'liga' off;
+  font-family: var(--font-family-body);
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 130%;
+  letter-spacing: 0.5px;
+  border: none;
+  padding: 0;
+  align-items: center;
+}
+
+.redesign-v2 a.button.primary:hover,
+.redesign-v2 button.primary:hover {
+  background-color: unset;
```

Comment:
I forgot [something from the styleguide](https://www.volvogroup.com/experiencesystem/en/#/brand/615bfcc428c818001105b4e3/61a0de32a7047974164f01dd/6231c4ba37a675408d8c3362). The hover and active state need to be changed. `text-decoration: underline;` needs to be added, and the colours changed.

Above in this file you can change `--c-cta-blue-hover` to have #2B8EDE as value and `--c-cta-blue-active` to have #104E9B as value. I checked the usage in the project, these changes will be fine.

Reply:
done